examples/react/getting-started/src/Product.tsx

```diff
@@ -0,0 +1,87 @@
+import algoliasearch from 'algoliasearch/lite';
+import { type Hit } from 'instantsearch.js';
+import React from 'react';
+import {
+  Configure,
+  Hits,
+  InstantSearch,
+  RelatedProducts,
+} from 'react-instantsearch';
+
+import './App.css';
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+
+export function Product({ pid }: { pid: string }) {
+  return (
+    <>
+      <header className="header">
+        <h1 className="header-title">
+          <a href="/">Getting started</a>
+        </h1>
+        <p className="header-subtitle">
+          using{' '}
+          <a href="https://github.com/algolia/instantsearch">
+            React InstantSearch
+          </a>
+        </p>
+      </header>
+
+      <div className="container">
+        <a href="/">← Back to search</a>
+        <InstantSearch
+          searchClient={searchClient}
+          indexName="instant_search"
+          insights={true}
+        >
+          <Configure hitsPerPage={1} filters={`objectID:${pid}`} />
+          <Hits
+            hitComponent={HitComponent}
+            classNames={{
+              root: 'ais-Hits--single',
+            }}
+          />
+          <RelatedProducts
+            itemComponent={ItemComponent}
+            emptyComponent={() => <></>}
```

Comment:
why?

Reply:
Without this I was getting a flash of "no results". Not sure if it should be avoided here or investigated and fixed first

Reply:
Maybe let's keep this here now and we can always remove it when we figure it out in the widget itself, wdyt?